dashboard/test/ui/features/teacher_dashboard.feature

```diff
@@ -163,6 +163,26 @@ Feature: Using the teacher dashboard
     # Wait for the thumbnail URL to be sent to the server.
     And I wait until element ".project_updated_at" contains text "Saved"
 
+    # Create a dance party project level and generate a thumbnail.
+
+    # We don't want to have to write the code by dragging blocks, so just remix
+    # an existing project-backed level, and then run the project.
```

Comment:
I'm fine with the approach you've chosen here, but FWIW we do have some tests that write code by dragging blocks, and others that inject block XML fixtures into the workspace.